README.md

```diff
@@ -32,6 +72,28 @@ Readmes on how to use each metric are available under the name `<name>_pc_readme
 
 Note that this distribution does not include the MATLAB Runtime, which must first be installed before running any of the compiled versions of these metrics. The runtime can be installed using `MCRInstaller.exe`, located [here](https://www.mathworks.com/products/compiler/matlab-runtime.html). Be aware that the compiled versions of the metrics were compiled in MATLAB R2013b (8.2). Future versions of MATLAB may cause the code not to run. This issue applies to both the compiled code and the source code. 
 
+## Iterated Nested Least Squares Algorithm (INLSA)
+
+The inlsa directory contains an iterated nested least-squares algorithm (INLSA) for fitting multiple data sets
+
+INLSA is an algorithm that allows multiple subjective datasets to be fitted to a single subjective scale. 
+INLSA computes the fit from a common set of objective metrics. 
+Files inlsa.m and pars_inlsa.m implement INLSA.
+File inlsa_demo.m creates made-up data for three (3) experiments and plots that data before and after running INLSA. The user can actually see what INLSA does to the data. Also the user gets a concrete example of how to call INLSA.  The user can just replace the made-up data with real data and the use INLSA.
```

Comment:
Suggest:
File inlsa_demo.m creates sample data for three (3) experiments and plots that data before and after running INLSA. The user can actually see what INLSA does to the data. Also the user gets a concrete example of how to call INLSA. The user can just replace the sample data with real data and then run INLSA.